packages/scxa-marker-gene-heatmap/src/clustersView/ClustersHeatmapView.js

```diff
@@ -90,16 +90,43 @@ class HeatmapView extends React.Component {
         isDisabled: ksWithMarkers ? !ksWithMarkers.includes(k.toString()) : false
       }))
 
+    const options = [{
+      label: `Number of clusters`,
+      options: kOptions
+    }]
+
+    if (metadata) {
+      const metadataOptions = metadata.map((metadata) => ({
+        ...metadata,
+        // only enable inferred cell type metadata option and disable the others like sex, cancer etc.
+        isDisabled: ![`Inferred cell type - authors labels`, `Inferred cell type - ontology labels`].includes(metadata.label),
```

Comment:
I would move these options to a variable and give them a descriptive name, then you could remove the comments in the previous line. The code would be cleaner with that change and more understandable for a new starter or even for us later when we are reading this code.
Thanks